src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Semantic/SemanticEffectRouter.ios.cs

```diff
@@ -26,6 +27,23 @@ protected override void Update(UIView view, SemanticEffectRouter effect)
 			else
 				view.AccessibilityTraits &= ~UIAccessibilityTrait.Header;
 
+			var a11yVisibility = SemanticEffect.GetA11yVisibility(Element);
+			switch (a11yVisibility)
+			{
+				case ImportantForA11y.Auto:
+					view.AccessibilityElementsHidden = false;
```

Comment:
I'm pretty sure the logic here for going back to Auto will also need to make sure it resets the `IsAccessibilityElement` property

Which you should be able to do like this

```C#
view.IsAccessibilityElement = view is UIControl
```